barquesubmit/client.go

```diff
@@ -0,0 +1,203 @@
+package barquesubmit
```

Comment:
Will there be other barque packages?

Reply:
in the barque repo. this is just a client, and we can't have a cycle.